src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -537,15 +554,11 @@ private static (Project project, ImmutableArray<Document> documents) GetProjectA
             return (project, documents);
         }
 
-        // make sure we always move remote workspace forward
-        private int _solutionVersion = 0;
-
-        private async Task UpdatePrimaryWorkspace(RemoteHostClient client, Solution solution)
+        private static async Task UpdatePrimaryWorkspace(RemoteHostClient client, Solution solution)
         {
-            var checksum = await solution.State.GetChecksumAsync(CancellationToken.None);
             await client.TryInvokeAsync<IRemoteAssetSynchronizationService>(
                 solution,
-                async (service, solutionInfo, cancellationToken) => await service.SynchronizePrimaryWorkspaceAsync(solutionInfo, checksum, _solutionVersion++, cancellationToken),
+                async (service, solutionInfo, cancellationToken) => await service.SynchronizePrimaryWorkspaceAsync(solutionInfo, cancellationToken),
```

Comment:
passing solutionInfo and checksum is entirely redundant (PinnedSolutionInfo has always contained the solution checksum).